src/test/java/easytests/api/v1/controllers/IssuesControllerTest.java

```diff
@@ -129,6 +137,65 @@ public void testListForbidden() throws Exception {
     /**
      * show(issueId)
      */
+    @Test
+    public void testShowSuccess() throws Exception {
+        final IssueModelInterface issueModel = new IssueModel();
+        issueModel.map(this.issueSupport.getEntityFixtureMock(0));
+        when(this.issuesOptionsBuilder.forAuth()).thenReturn(new IssuesOptions());
+        when(this.issuesService.find(any(Integer.class))).thenReturn(issueModel);
+        when(this.acl.hasAccess(any(SubjectModelInterface.class))).thenReturn(true);
+
+        this.mvc.perform(get("/v1/issues/1")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andExpect(content().contentTypeCompatibleWith(MediaType.APPLICATION_JSON))
+                .andExpect(content().json(new JsonSupport()
+                        .with(id, issueModel.getId())
+                        .with(name, issueModel.getName())
+                        .build()
+                ))
+                .andReturn();
+    }
+
+    @Test
+    public void testShowNotFound() throws Exception {
+        when(this.issuesOptionsBuilder.forAuth()).thenReturn(new IssuesOptions());
+        when(this.issuesService.find(any(Integer.class), any(IssuesOptionsInterface.class))).thenReturn(null);
+
+        mvc.perform(get("/v1/issues/1")
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isNotFound())
+                .andExpect(content().string(""))
+                .andReturn();
+    }
+
+    @Test
+    public void testShowWithQuizesSucces() throws Exception {
```

Comment:
there is no need to test that case, it is not differ from the `testShowSuccess` one